plugins/modules/aci_l3out_floating_svi_path_secondary_ip.py

```diff
@@ -0,0 +1,418 @@
+                    "nameAlias": "",
+                    "ownerKey": "",
+                    "ownerTag": ""
+                }
+            }
+        }
+    ]
+proposed:
+  description: The assembled configuration from the user-provided parameters
+  returned: info
+  type: dict
+  sample:
+    {
+        "fvTenant": {
+            "attributes": {
+                "descr": "Production environment",
+                "name": "production"
+            }
+        }
+    }
+filter_string:
+  description: The filter string used for the request
+  returned: failure or debug
+  type: str
+  sample: ?rsp-prop-include=config-only
+method:
+  description: The HTTP method used for the request to the APIC
+  returned: failure or debug
+  type: str
+  sample: POST
+response:
+  description: The HTTP response from the APIC
+  returned: failure or debug
+  type: str
+  sample: OK (30 bytes)
+status:
+  description: The HTTP status from the APIC
+  returned: failure or debug
+  type: int
+  sample: 200
+url:
+  description: The HTTP url used for the request to the APIC
+  returned: failure or debug
+  type: str
+  sample: https://10.11.12.13/api/mo/uni/tn-production.json
+"""
+
+from ansible_collections.cisco.aci.plugins.module_utils.aci import ACIModule, aci_argument_spec, aci_annotation_spec
+from ansible.module_utils.basic import AnsibleModule
+
+
+def main():
+    argument_spec = aci_argument_spec()
+    argument_spec.update(aci_annotation_spec())
+    argument_spec.update(
+        tenant=dict(type="str", aliases=["tenant_name"], required=True),
+        l3out=dict(type="str", aliases=["l3out_name"], required=True),
+        node_profile=dict(type="str", aliases=["node_profile_name", "logical_node"], required=True),
+        interface_profile=dict(type="str", aliases=["interface_profile_name", "logical_interface"], required=True),
+        state=dict(type="str", default="present", choices=["absent", "present", "query"]),
+        pod_id=dict(type="str", required=True),
+        node_id=dict(type="str", required=True),
+        encap=dict(type="str", required=True),
+        address=dict(type="str", aliases=["addr", "ip_address"]),
+        floating_ip=dict(type="str", aliases=["floating_address"]),
+        domain_type=dict(type="str", choices=["physical", "vmware"], required=True),
+        domain=dict(type="str", required=True),
+        secondary_ip=dict(type="str", aliases=["secondary_floating_address"]),
+        access_encap=dict(type="str"),
+    )
+
+    module = AnsibleModule(
+        argument_spec=argument_spec,
+        supports_check_mode=True,
+        required_if=[
+            [
+                "state",
+                "absent",
+                [
+                    "secondary_ip",
+                ],
+            ],
+            [
+                "state",
+                "present",
+                [
+                    "secondary_ip",
+                ],
+            ],
+        ],
+    )
```

Comment:
why again a different format?
```suggestion
        required_if=[
            ["state", "absent", ["secondary_ip"]],
            ["state", "present", ["secondary_ip"]],
        ],
    )
```

Reply:
Not really sure what happened here. I think I mistyped the python black formatting command with the wrong line limiter perhaps? Changed it.